examples/07-EMIT/ComputeInterferenceType.py

```diff
@@ -69,7 +68,7 @@ def install(package):
 domain = emitapp.results.interaction_domain()
 
 if tx_radios is None or rx_radios is None:
-    print("No recievers or transmitters in design.")
+    print("No receivers or transmitters in design.")
```

Comment:
```suggestion
    print("No receivers or transmitters are in the design.")
```
Message strings should be complete sentences.